empiric-package/empiric/core/entry.py

```diff
@@ -1,21 +1,22 @@
 from collections import namedtuple
+from typing import Any, List
 
 from empiric.core.utils import str_to_felt
 
 Entry = namedtuple("Entry", ["key", "value", "timestamp", "source", "publisher"])
 
 
-def serialize_entry(entry):
+def serialize_entry(entry) -> List[Any]:
     return [entry.key, entry.value, entry.timestamp, entry.source, entry.publisher]
 
 
-def serialize_entries(entries):
+def serialize_entries(entries) -> List[Any]:
     expanded = [serialize_entry(entry) for entry in entries]
     flattened = [x for entry in expanded for x in entry]
     return [len(entries)] + flattened
 
 
-def construct_entry(key, value, timestamp, source, publisher):
+def construct_entry(key, value, timestamp, source, publisher) -> Entry:
```

Comment:
Not sure if this is already the case, but should type it such that the namedtuple elements in Entry are all ints

Reply:
Just changed it. I think we should also consider making `Entry` a class.